client/my-sites/site-settings/form-general.jsx

```diff
@@ -794,6 +837,10 @@ const getFormSettings = ( settings ) => {
 
 		wpcom_coming_soon: settings.wpcom_coming_soon,
 		wpcom_public_coming_soon: settings.wpcom_public_coming_soon,
+		wpcom_gifting_subscription:
+			typeof settings.wpcom_gifting_subscription === 'undefined'
```

Comment:
This also coerces the value into a boolean, suggesting it since it's shorter.
```
wpcom_gifting_subscription: !! settings.wpcom_gifting_subscription
```

Reply:
Thanks 🎉